nautilus_trader/adapters/databento/data_utils.py

```diff
@@ -221,20 +221,60 @@ def databento_data(
 
 
 def save_data_to_catalog(
-    definition_file,
-    data_file,
     *folders,
+    definition_file=None,
+    data_file=None,
     base_path=None,
     write_data_mode="overwrite",
 ):
+    """
+    Save Databento data to a catalog.
+
+    This function loads a catalog, processes Databento definition and data files,
+    and writes the processed data to the catalog.
+
+    Parameters
+    ----------
+    *folders : str
+        Variable length argument list of folder names to be used in the catalog path.
+    definition_file : str or Path, optional
+        Path to the Databento definition file.
+    data_file : str or Path, optional
+        Path to the Databento data file.
+    base_path : str or Path, optional
+        Base path for the catalog.
+    write_data_mode : str, optional
+        Mode for writing data to the catalog. Default is "overwrite".
+
+    Returns
+    -------
+    dict
+        A dictionary containing:
+        - 'catalog': The loaded catalog object.
+        - 'nautilus_definition': Processed Databento definition data.
```

Comment:
Do we need the `'nautilus_'` prefix for these?

Reply:
Yes it means nautilus objects as opposed to databento objects. It's to make things clear as both types of objects are returned by the databento_data function.